examples/basic/print.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
Would be nice to include nested subcommands example as well

Reply:
Added a new directory with nested subcommands, as well as one with dynamic manipulation.